toolkit/tools/internal/network/network_test.go

```diff
@@ -57,8 +82,8 @@ func TestDownloadFile(t *testing.T) {
 		{
 			name: "TestDownloadWithCancel",
```

Comment:
Rename to `TestDownloadWithContext`? The current name suggests that you call cancel.